docs/book/src/cronjob-tutorial/testdata/project/internal/controller/cronjob_controller_test.go

```diff
@@ -129,7 +129,7 @@ var _ = Describe("CronJob controller", func() {
 			By("By checking the CronJob has zero active Jobs")
 			Consistently(func(g Gomega) {
 				g.Expect(k8sClient.Get(ctx, cronjobLookupKey, createdCronjob)).To(Succeed())
-				g.Expect(createdCronjob.Status.Active).To(HaveLen(0))
+				g.Expect(createdCronjob.Status.Active).To(HaveLen(0)) // nolint:ginkgolinter
```

Comment:
hey @jongwooo 

to apply changes to these files, you need to change code in `hack/docs/internal` folder, and then run `make generate`.

For example, for this line, you need to change it here https://github.com/kubernetes-sigs/kubebuilder/blob/9c36b61aaa12c99135b3c54e25a5864ab83925cf/hack/docs/internal/cronjob-tutorial/writing_tests_controller.go#L151